app/assets/stylesheets/views/_coronavirus_manage_pages.scss

```diff
@@ -1,6 +1,5 @@
-.covid-manage-page__accordion-summary-list,
-.covid-manage-page__timeline-entry-summary-list,
-.covid-manage-page__announcements-summary-list {
+.covid-manage-page__summary-list,
```

Comment:
Can the commit note be expanded to explain why these changes are being made. Thanks.